housekeeper/archive/core.py

```diff
@@ -35,5 +38,6 @@ def get_archiveassets(run_id):
     for archive_type in ("data", "result"):
         category = "archive-{}".format(archive_type)
         query = api.assets(run_id=run_id, category=category)
-        asset = query.first()
-        yield archive_type, asset
+        assets = query.all()
+        for asset in assets:
+            yield archive_type, asset
```

Comment:
When encrypting something, you get two files: the encrypted tar archive and the gpg key. This query will return both of those as I tag them both with `archive-{data,result}`.
So I'm unsure if `archive-{}` should only have one result.